app/src/main/java/com/weatherxm/ui/components/HeaderView.kt

```diff
@@ -0,0 +1,53 @@
+package com.weatherxm.ui.components
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import com.weatherxm.R
+import com.weatherxm.databinding.ViewHeaderBinding
+import org.koin.core.component.KoinComponent
+
+open class HeaderView : LinearLayout, KoinComponent {
```

Comment:
Are we sure that the header should be so big? I thought the title should be inside the scrolling area

Reply:
Not sure really, I took it from the designs, and I thought due to the fact that it's a header, that it won't be scrollable. Let's discuss it internally with the product team.